bezier_library/src/bezier_library.cpp

```diff
@@ -81,6 +82,23 @@ bool Bezier::getTranslationMode()
   return use_translation_mode_;
 }
 
+void Bezier::SetSurfacingOn(){
+  ROS_INFO_STREAM("WARM : Running in surfacing mode");
```

Comment:
In this library we always prefix our messages with the name of the member that prints a message; like here:
https://github.com/ros-industrial-consortium/bezier/blob/indigo-devel/bezier_library/src/bezier_library.cpp#L166